### PR TITLE
update releases to v3.18

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

**Comment by amyrlam** [comment]
Hm... not sure about the Lighthouse error yet, or why /ember-community-survey-2019.html is in the Percy diff. https://travis-ci.org/github/ember-learn/ember-website/builds/683546010#L800-L824

File: data/project/ember/beta.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ filter:
   - /ember\./
   - /ember-template-compiler/
 repo: emberjs/ember.js
-initialVersion: 3.17.0 # Manually update, see https://libraries.io/npm/ember-source throughout
-lastRelease: 3.18.0-beta.5 # Manually update
-futureVersion: 3.18.0-beta.5 # Manually update
-finalVersion: 3.18.0 # Manually update
+initialVersion: 3.18.0 # Manually update, see https://libraries.io/npm/ember-source throughout
+lastRelease: 3.19.0-beta.3 # Manually update
+futureVersion: 3.18.0-beta.4 # Manually update
```
Comment (locks) [inline data/project/ember/beta.md:10]:
```suggestion
futureVersion: 3.19.0-beta.4 # Manually update
```

**Comment by locks** [comment]
There's definitely some weirdness going on in Percy:
image

I am also not familiar with the lighthouse part, we may need some insight from @mansona :)

**Comment by mansona** [comment]
@locks so it turns out that this issue is because the version of Chrome that is installed on Travis is **super** old... which is odd because we specifically want the `stable` version of Chrome 🤔 

But after a little digging, it turns out that the Chrome install failed for that Travis run with the following message: 

```
$ wget --no-verbose -O /tmp/$(basename $CHROME_SOURCE_URL) $CHROME_SOURCE_URL
2020-05-05 21:11:10 URL:https://dl.google.com/dl/linux/direct/google-chrome-stable_current_amd64.deb [66985784/66985784] -> "/tmp/google-chrome-stable_current_amd64.deb" [1]
dpkg-deb: error: archive '/tmp/google-chrome-stable_current_amd64.deb' has premature member 'control.tar.xz' before 'control.tar.gz', giving up
dpkg: error processing archive /tmp/google-chrome-stable_current_amd64.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 /tmp/google-chrome-stable_current_amd64.deb
```

I tried restarting the job and it looks like travis is still having the same issue 🤷 I'll do some more digging

**Pull request merged**